.travis.yml

```diff
@@ -8,3 +8,6 @@ install:
     - pip install -r requirements.txt
     - pip install -r test-requirements.txt
 script: flake8
+before_install: pip install codecov
+after_success: codecov
```

Comment:
Let's use the bash uploader: https://docs.codecov.io/v4.3.0/docs/about-the-codecov-bash-uploader
This way we don't need to `pip install codecov`.